.github/workflows/download-link-check-schedule.yml

```diff
@@ -1,26 +1,53 @@
-name: DVC Download Link Check
+name: Scheduled DVC Download Link Check
 
 on:
+  pull_request:
   schedule:
     - cron: '0 0-23 * * *'
 
+concurrency:
+  group: scheduled-download-link-check-${{ github.head_ref }}
+  cancel-in-progress: true
+
 jobs:
   run:
-    name: Initialize
+    name: Check Download Links
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Download Link Checker
+
+      - name: Run Lychee
         uses: lycheeverse/lychee-action@v1.0.8
         with:
           args:
             --verbose https://dvc.org/ --base https://dvc.org/ --include
             /download/
         env:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
-      - name: Create Issue From File
-        uses: peter-evans/create-issue-from-file@v3
+
+      - name: Find latest open Link Check issue
+        uses: actions/github-script@v6
+        id: set-result
+        with:
+          script: |
+            const issues = await github.rest.issues.listForRepo({
+              owner: context.payload.repository.owner.login,
+              repo: context.payload.repository.name,
+              state: "open",
+              creator: "github-actions[bot]",
+              sort: "created"
+            })
+            const latestLinkCheckIssue = issues.data.find(
+              (issue)=>(
+                issue.title === "DVC Download Link Checker Report"
+              )
+            )
+            if(latestLinkCheckIssue) return latestLinkCheckIssue.number
```

Comment:
Couldn't find an action that finds Issues specifically, so `github-script`'s rapid prototyping comes to the rescue.
We can generalize this in the future if needed, but maybe a reusable workflow will suffice.